### PR TITLE
feat: overview architecture SVG schema

### PR DESCRIPTION
**Pull request by kpanot**
## Proposed change

Architecture overview

### CODE REVIEW

File: assets/schemas/otter-environment.drawio.svg
Comment (matthieu-crouzet) [inline assets/schemas/otter-environment.drawio.svg:1]: Typo for the blue legend :)

Comment (matthieu-crouzet) [reply assets/schemas/otter-environment.drawio.svg:1]: Arrow from `@o3r/testing` to `@o3r/core` pass behind the box title `Otter core`

Comment (matthieu-crouzet) [reply assets/schemas/otter-environment.drawio.svg:1]: Small issue with the arrow from customization to core

Comment (matthieu-crouzet) [reply assets/schemas/otter-environment.drawio.svg:1]: Arrow from customization to hosting goes through mobile that's a bit strange

Comment (vscaiceanu-1a) [reply assets/schemas/otter-environment.drawio.svg:1]: `@o3r/crete` should be an initializer?

**Review by matthieu-crouzet — APPROVED** [review]

**Review by vscaiceanu-1a — APPROVED** [review]

**Pull request merged**